integrations/huggingface-transformers/tutorials/sentiment-analysis/sentiment-analysis-cli.md

```diff
@@ -0,0 +1,328 @@
+  --distill_teacher disable \
+  --train_file ./rotten_tomatoes-train.csv --validation_file ./rotten_tomatoes-validation.csv  --input_column_names text --label_column_name label \
+  --output_dir sparse_quantized_bert-text_classification_rotten_tomatoes-local_dataset --max_seq_length 128 --per_device_train_batch_size 32 \
+  --per_device_eval_batch_size 32 --preprocessing_num_workers 6 --do_train --do_eval --evaluation_strategy epoch --fp16  \
+  --save_strategy epoch --save_total_limit 1
+```
+
+### Sparse Transfer Learning with a Custom Teacher (Rotten Tomatoes)
+
+To support the transfer learning process, we can apply model distillation, just like we did for the SST2 case.
+You are free to use the native Hugging Face workflows to train the dense teacher model (and can even
+pass a Hugging Face model identifier to the command), but you can also use the SparseML CLI as well. 
+
+#### Train The Dense Teacher
+
+Run the following to train a dense model on Rotten Tomatoes:
+```
+sparseml.transformers.train.text_classification \
+  --output_dir dense-teacher_rotten_tomatoes \
+  --model_name_or_path zoo:nlp/masked_language_modeling/bert-base/pytorch/huggingface/wikipedia_bookcorpus/base-none \
+  --recipe zoo:nlp/sentiment_analysis/bert-base/pytorch/huggingface/sst2/base-none \
+  --recipe_args '{"init_lr":0.00003}' \
+  --dataset_name rotten_tomatoes --input_column_names "text" --label_column_name "label" \
+  --max_seq_length 128 --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16  \
+  --save_strategy epoch --save_total_limit 1
+```
+
+The model converges to ~86.9% accuracy without any hyperparameter search.
+
+Note that used the dense version of BERT (the stub ends in `base-none`) as the starting point for the training 
+and passed a recipe from [SparseZoo](https://sparsezoo.neuralmagic.com/models/nlp%2Fsentiment_analysis%2Fbert-base%2Fpytorch%2Fhuggingface%2Fsst2%2Fbase-none) which was used to train the 
+dense teacher for the SST2 task. Since the SST2 task is similiar to the Rotten Tomatoes task, these parameters are a solid
+place to start. This recipe contains no sparsity related modifiers and only controls the learning rate and number of epochs. As such, the script
+will run typical fine-tuning, resulting in a dense model.
+
+Here's what the recipe looks like:
+```yaml
+version: 1.1.0
+
+# General Variables
+num_epochs: 8
+init_lr: 1.5e-4 
+final_lr: 0
+
+# Modifiers:
+training_modifiers:
+  - !EpochRangeModifier
+      end_epoch: eval(num_epochs)
+      start_epoch: 0.0
+    
+  - !LearningRateFunctionModifier
+    start_epoch: 0
+    end_epoch: eval(num_epochs)
+    lr_func: linear
+    init_lr: eval(init_lr)
+    final_lr: eval(final_lr)
+```
+
+#### Sparse Transfer Learning with a Custom Teacher
+
+With the dense teacher trained, we can sparse transfer learn with the help of the teacher by passing
```

Comment:
With the dense teacher trained, we can sparse transfer ~~learn~~ with the help of the teacher by passing